src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,161 @@
+  // Template Enums
+  readonly NameRequestStates = NameRequestStates
+
+  @Prop()
+  private nameRequest: any
+
+  private parsedNameRequest: NameRequestIF
+
+  private nameRequestDetails:NameRequestDetailsIF
+
+  private nameRequestApplicant: NameRequestApplicantIF
+
+  private created () {
+    this.parsedNameRequest = this.generateNameRequestState(this.nameRequest, null)
+    this.nameRequestDetails = this.parsedNameRequest.details
+    this.nameRequestApplicant = this.parsedNameRequest.applicant
+  }
+
+  /** The entity title  */
+  private entityTypeDescription () :string {
+    const entityType = this.parsedNameRequest.entityType
+    if (entityType === 'BC') {
+      return 'Benefit Company'
+    } else if (entityType === 'CP') {
+      return 'Cooperative Association'
+    }
+    return ''
+  }
+
+  /** The request type */
+  private requestType () : string {
+    return 'New Business'
+  }
+
+  /** Return formatted expiration date */
+  private formattedExpirationDate () : string {
+    return this.toReadableDate(this.nameRequestDetails.expirationDate)
+  }
+
+  /** Return consent received string by checking if conditional and if consent has been received */
+  private conditionConsent (): string {
+    if (this.nameRequestDetails.status === NameRequestStates.CONDITIONAL) {
+      return this.nameRequestDetails.consentFlag === 'R' ? 'Received' : 'Not Received'
+    } else {
+      return 'Not Required'
+    }
+  }
+
+  /** Return formatted applicant name */
+  private applicantName () : string {
+    let name = this.nameRequestApplicant.firstName
+    if (this.nameRequestApplicant.middleName) {
+      name = `${name} ${this.nameRequestApplicant.middleName} ${this.nameRequestApplicant.lastName}`
+    } else {
+      name = `${name} ${this.nameRequestApplicant.lastName}`
+    }
+    return name
+  }
+
+  /** Return formatted address string */
+  private applicantAddress () : string {
+    // Get Address info
+    const city = this.nameRequestApplicant.city
+    const stateProvince = this.nameRequestApplicant.stateProvinceCode
+    const postal = this.nameRequestApplicant.postalCode
+    const country = this.nameRequestApplicant.countryTypeCode
+      ? getName(this.nameRequestApplicant.countryTypeCode) : ''
+
+    // Build address lines
+    let address = this.nameRequestApplicant.addressLine1
+    if (this.nameRequestApplicant.addressLine2) {
+      address = `${address}, ${this.nameRequestApplicant.addressLine2}`
+    }
+    if (this.nameRequestApplicant.addressLine3) {
+      address = `${address}, ${this.nameRequestApplicant.addressLine3}`
+    }
+
+    return `${address}, ${city}, ${stateProvince}, ${postal}, ${country}`
+  }
+
+  private formatStatusText (status: string) : string {
+    return status.charAt(0).toUpperCase() + status.slice(1).toLowerCase()
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/theme.scss";
```

Comment:
FYI, this import is only needed if you use some of its CSS variables.

Reply:
Removed the import